### PR TITLE
Recall Results: 

### LINKED ISSUE #26: Create Recall Results HTML page
Create a page that extends Recall  information to replace the current input html page

### LINKED ISSUE #37: Correcting app.py function calls
Error in building function calls:

Correcting:

image

### PR DESCRIPTION
**Pull request by Mr-3rd**
Errors in calling modules from app.py, need help

### CODE REVIEW

**Comment by Mr-3rd** [comment]
@amylynndee & @pengxuemoua  I'm struggling to satisfy our API calls  to begin looping and harvesting data for the recalls page

**Comment by Mr-3rd** [comment]
@claraj, here is my pull for the module is not callable error I am receiving

the code runs but the API block of code in app.py refuses to call from the files

**Comment by Mr-3rd** [comment]
@claraj, @amylynndee , @pengxuemoua  I got the code to work in the most recent update, now I need to start building html that will create the html elements we want

**Review by pengxuemoua — APPROVED** [review]
Code looks good! Approved and ready to merge @Mr-3rd

File: templates/car_recalls.html [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+            Affected Component: {{ recall.Component  }}
+            <br>
+            Summary of recall: {{ recall.Summary }}
+        </li>
+    {% endfor %}
+    </ul>
+{% endif %}
+
+<br>
+{% for video in car_videos %}
```
Comment (Mr-3rd) [inline templates/car_recalls.html:26]: @claraj, why would this loop only produce 1 item when the loop will have one for each component?

File: apis/video_api.py [hunk 2]
```diff
@@ -72,10 +81,4 @@ def get_car_videos(car , recalls):
             error = 'A catastrophic error has occurred'
             logging.exception(error)
             return error
-        
-    # if there are no videos returned in the list return a string with  that message
-    if len(video_links) == 0:
```
Comment (Mr-3rd) [inline apis/video_api.py:77]: @claraj  I did move this to within the try error loop to make it match the photo api call

**Review by Mr-3rd — COMMENTED** [review]
@claraj  I added some notes about the areas I seem to be experiencing a problem.

This is the list of API keys required:

YOUTUBE_API
FLICKR_API
YELP_API

File: apis/video_api.py [hunk 3]
```diff
@@ -59,6 +62,12 @@ def get_car_videos(car , recalls):
                 # append both to the list
                 video_links.append({'title': title, 'embed': embed})
 
+                # if there are no videos returned in the list return a string with  that message
+            if len(video_links) == 0:
+                return 'Car recall videos not found'
+            else:
+                return video_links
```
Comment (claraj) [inline apis/video_api.py]: This return statement is inside the loop, so returns after the first recall has been processed

Comment (Mr-3rd) [reply apis/video_api.py]: Addressed in recent commit

**Review by pengxuemoua — APPROVED** [review]
Code looks great! Approved!

**Comment by amylynndee** [comment]
Code looks good, this is approved to merge!

**Review by amylynndee — APPROVED** [review]
Approved.

**Pull request merged**